projects/anti_scaling/distillation.py

```diff
@@ -338,6 +341,9 @@ def _perform_forward_passes(self, batch: Batch) -> ForwardPassOutputs:
         mask = self._manipulate_mask(
```

Comment:
nit: maybe `mask` should now be `score_mask` to differentiate it from `decoder_mask`?